Makefile

```diff
@@ -13,8 +13,9 @@ test-portus: build
 
 test-ipc: build
 ifeq ($(OS), Linux)
-	sudo ./target/debug/nltest
-	sudo ./target/debug/kptest
+	sudo ./target/debug/ipc_latency -i 10 --impl unix
+	sudo ./target/debug/ipc_latency -i 10 --impl nl
+	sudo ./target/debug/ipc_latency -i 10 --impl kp
```

Comment:
What is the story on nltest/kptest? Is it going away? (seems to be, since it's removed here)

This commit should probably remove those files and have some explanatory text in the commit message notes saying how this is tested now.

Reply:
I haven't made any changes to kptest, but I looked at kptest and it seems to do something very similar to what ipc_latency does when running the kp benchmark. I figured if we're using ipc_latency for nltest, we might as well use it for kptest too unless there's a good reason not to.

Reply:
If you add functionality to test the character device (kp) in ipc_latency then it's fine to remove kptest, but otherwise I wouldn't remove it until then. We still need something to test whether or not the character device works.

Reply:
I'll move the kptest stuff to a different commit, since I nothing else I changed actually affects the kptest.